### PR TITLE
Emphasizing the difference between this project and endroid/qr-code

### PR DESCRIPTION
**Pull request by ThomasLandauer**
Since this is the thing that changed from smhg/sepa-qr, I think it's better to show what this library now actually does (and what it does not :-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,8 +55,8 @@ Set the service tag. Currently (?) only one value is allowed: BCD.
 ### setVersion($version = 2)
 Set the SEPA QR standard version. In version 1 a BIC is mandatory. In version 2 a BIC is only mandatory outside EEA countries.
 
-### setCharacterSet($characterSet = SepaQr::UTF_8)
-Set the character set. Available constants are **UTF_8**, **ISO8859_5**, **ISO8859_1**, **ISO8859_7**, **ISO8859_2**, **ISO8859_10**, **ISO8859_4** or **ISO8859_15**.
+### setCharacterSet($characterSet = SepaQrData::UTF_8)
```
Comment (smhg) [inline README.md:58]: Good catch, thank you!

File: README.md [hunk 2]
```diff
@@ -1,32 +1,48 @@
...
-$sepaQrData
-  ->setName('Name of the beneficiary')
-  ->setIban('BE123456789123456789')
-  ->setAmount(100); // The amount in Euro
-
 $result = Builder::create()
     ->writer(new PngWriter())
     ->data($sepaQrData) // calls $sepaQrData->__toString()
-    ->errorCorrectionLevel(new ErrorCorrectionLevelMedium())
+    ->errorCorrectionLevel(new ErrorCorrectionLevelMedium()) // "medium" is the standard for EPC QR codes, but endroid/qr-code's default is "low"
```
Comment (smhg) [inline README.md:45]: Good to explain this, thanks.

**Review by smhg — CHANGES_REQUESTED** [review]
I suggest to only add the last 2 changes in this PR (L29 + L42-L43). They are definitely necessary.

Your other changes just present the same problem in a different way if you ask me. I definitely agree it should be made clear that a) the output of this project is an intermediary step and b) you need endroid/qr-code (or similar). But this needs a different structure of the readme rather than the changes you present I think. I'll put that in an issue.
Would you be ok to undo most of your work?

**Comment by ThomasLandauer** [comment]
The (by far) most important thing in README is this:
```
BCD
002
1
SCT
Name of the beneficiary
BE123456789123456789
EUR100.00
```
Cause this is what this library actually does!

I'm questioning your "intermediary step" point of view, from two sides:
* If somebody writes this text manually, they don't need this library at all; they can throw their text directly into any QR library.
* Other use-cases are possible as well, e.g. verification of an existing QR code: Use some decoding library and compare its output to this library's output. Or do something else with this text; that's up to the user.

So in total: Showing this codeblock is a must, all the rest (i.e. hints for endroid etc.) is a plus ;-)

**Comment by smhg** [comment]
> * If somebody writes this text manually, they don't need this library at all; they can throw their text directly into any QR library.

Absolutely. However, the whole point of this library is that you don't need to know about, specifically, this format's existence. You want to turn your data (amount, iban,...) into a _valid_ QR code. This library takes care of validation and formatting for you and you print your QR renderer.

> * Other use-cases are possible as well, e.g. verification of an existing QR code: Use some decoding library and compare its output to this library's output. Or do something else with this text; that's up to the user.
> 
> So in total: Showing this codeblock is a must, all the rest (i.e. hints for endroid etc.) is a plus ;-)

Let's agree to disagree on this :-)

As a (first-time) user, I want a (one-line?) description + installation instructions (?) + how I use it. And that last part means piping the output into a QR generating library. I really don't need to see what it does with my data (amount, iban,...) _initially_. I am fine with explaining it in a more in-depth section.

It makes sense to show the simplest route to success first, no?

**Comment by ThomasLandauer** [comment]
> It makes sense to show the simplest route to success first, no?

* In a perfect world, maybe :-) If I need two libraries for what looks like one task, I want to get some understanding what each of them does, since (sooner or later) things won't work as expected anymore...
* The absolutely simplest route to success might be to not installing it at all - just copy the text manually and throw it into endroid! ;-) So I would even go so far and explicitly list this as a valid (no)use case. In the end, the real power of this library is easier data generation if the included information (amount, iban,...) changes often.

> I am fine with explaining it in a more in-depth section.

Well, since you don't have such an "in-depth section", it comes down to the question if you show "my" codeblock a few lines above or below "your" endroid instructions ;-) I'm sure we can find a compromise for this...

**Pull request closed without merging**